deploy/docker/cp-api-srv/init-api

```diff
@@ -126,6 +126,11 @@ function sign_and_publish_pipe_win_distribution() {
     rm -rf pipe
 }
 
+function unpack_monitoring_template() {
+    local jar_path="$1"
+    unzip -p "$jar_path" static/monitoring_report_template.xls > "$CP_API_MONITORING_XLS_TEMPLATE_PATH"
```

Comment:
I'm not exactly sure but the path in the jar seems to be `BOOT-INF/classes/static/monitoring_report_template.xls`. Could you please doublecheck it?